service/webodm-docker.service

```diff
@@ -8,6 +8,7 @@ Type=forking
 User=odm
 Group=odm
 WorkingDirectory=/opt/WebODM
+Environment=PATH=/home/odm/.local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
This insures that the service uses the version of docker-compose installed by pip, which is much newer than the version available from Ubuntu repositories.